Umbraco-Cloud/release-notes/2022-01-06-ReleaseNote.md

```diff
@@ -0,0 +1,28 @@
+# January 6, 2022  Release Notes
+
+_Break project reference + See project information + Various improvements_
+
+### Key Takeaways
+- **Break reference between baseline and child project** - As an admin you can now break the reference between a baseline and child projects
+- **New Project Overview Page** - See the project information such as plan, payment status for your project on the new project overview page.
+- **Various tweaks and improvements** - We have since December provided a lot of small changes and improvements.
+
+### Break reference between baseline and child project
```

Comment:
Looks good, could you link the headline to this page where we have added the information about breaking the baseline:
[https://our.umbraco.com/documentation/Umbraco-Cloud/Getting-Started/Baselines/Break-baseline/]